test/test_gnoi.py

```diff
@@ -22,7 +22,7 @@ def test_gnoi_reboot(self):
         assert new_cnt == old_cnt+1, 'DBUS API is not invoked'
 
     def test_gnoi_reboot_halt(self):
-        ret, old_cnt = gnmi_dump('DBUS config reload')
```

Comment:
I see test case comes from the last commit. Maybe consider squashing this one into the last one?

Reply:
I don't have write permission to the repo for squash merge unfortunately.